_posts/2021-01-22-sars-cov-2-variant-analysis.md

```diff
@@ -0,0 +1,60 @@
+
+While the UK's COVID-19 Genomics Consortium (COG-UK) has been pioneering such
+monitoring since early on in the pandemic, other countries are now eager to
+establish their own networks for genomic surveillance.
+
+In the framework of [covid19.galaxyproject.org](https://covid19.galaxyproject.org), an international team of Galaxy
+developers and researchers from different disciplines (including members of the
+Galaxy EU team and the [Institute of Virology of the University Medical Center Freiburg](https://www.uniklinik-freiburg.de/virologie-en.html))
+have collaborated since February last year to establish a set of workflows for
+SARS-CoV-2 genomics analysis - including the ones that are urgently needed now.
+
+The European Galaxy server offers these as
+[__public workflows__](https://usegalaxy.eu/workflows/list_published).
+Just search the list for *covid19.galaxyproject.org* and you will find our
+recommended, best-practice workflows for SARS-CoV-2 variant analysis from
+a variety of sequencing data sources: be it whole-genome sequencing or
+ampliconic (ARTIC) data sequenced using short-read (Illumina, single-end or
+paired-end) or long-read (ONT) data, we have workflows for calling variants
+(even low-frequency intra-sample ones), generating standardized
+technology-independent reports, and for building reliable consensus sequences!
+
+All these workflows have been built and tested specifically with scalability in
+mind: on [usegalaxy.eu](https://usegalaxy.eu) we have verified, using large batches of COG-UK and other
+data, that they let you analyze even thousands of samples in parallel.
+
+Combine, *e.g.*, our
+[__ARTIC Illumina PE variant calling__](https://usegalaxy.eu/u/wolfgang-maier/w/covid19-variation-analysis-on-artic-pe)
+workflow with the workflow for
+[__consensus sequence building__](https://usegalaxy.eu/u/wolfgang-maier/w/covid-19-consensus-construction)
+and you have an analysis pipeline for analyzing typical diagnostic batches of
+data, like most of the data generated by COG-UK, and for turning the results
+into a set of consensus sequences ready for submission to
+[GISAID](https://www.gisaid.org) or for [clade assignment with Nextstrain](https://clades.nextstrain.org).
+
+If, instead of reducing all the precise variant-calling information into a
+simplifying consensus sequence, you would like to explore the full picture of
+potential intra-sample variation, you could run our
+[__variation reporting__](https://usegalaxy.eu/u/wolfgang-maier/w/covid19-variation-analysis-reporting)
+workflow, which will create different graphical and tabular views on the
+variant data, which you can use as a starting point for further investigation,
+*e.g.* with [__Interactive Notebooks__](https://live.usegalaxy.eu) or using resources like
+https://observablehq.com/collection/@spond/intrahost-variant-exploration.
+
+However, to explore the entire picture of the SARS-CoV-2 genome, RAW sequencing data is needed. Therefore, we stronlgy recommend,
```

Comment:
@wm75 what do you think about the following text?

I think one or two images pept das ganze noch auf :)